src/electron/office/document_client.cc

```diff
@@ -5,14 +5,18 @@
 #include "office/document_client.h"
```

Comment:
Should cleanup unused headers

Reply:
> Worth noting that this doesn't support copying from the web or MS Word or Google Docs, formatting doesn't get copied. This might be "good enough" for now, but definitely isn't what most people expect.
> 
> @skececi Will and I have discussed the alternative approach, but it's more intensive than this one

I just tested and I was able to copy this exact github message and paste it into the editor. I can also copy from the editor and paste into this github comment here: `GOING TO TEST COPYING IMAGE????jjjjj
kjkjkjkkj`

Reply:
done